text-utilities.h

```diff
@@ -36,6 +36,14 @@ static void set_font(struct pango_source *src, PangoLayout *layout) {
 			FcFontSet *font_set = FcFontSetCreate();
 			int count = FcFreeTypeQueryAll(src->font_file, -1, NULL, &count, font_set);
 			if (count > 0 ) {
+#if defined(PANGO_VERSION_CHECK) && PANGO_VERSION_CHECK(1,38,0)
```

Comment:
Even though the issue only affects 1.47 and above, I used 1.38 for this directive because it's the first version where the `pango_fc_font_map_config_changed` method was added.

Reply:
Is there a platform with such an old pango version still around? If not id prefer we do not perform this check.

Reply:
Good question. From a quick glance, CentOS \<7.4 has a sufficiently old version of pango, but I'd assume people would have upgraded to a later point release for that.

Reply:
Realized, after getting some sleep, a more relevant question: are there any distros that use Pango \< 1.38 but fontconfig \>= 2.12.91? (That is, can the pango directive be false, but the outer fontconfig directive be true?)

For RHEL/CentOS, the first version with a fontconfig version \>= 2.12.91 is 7.6; previous versions used fontconfig 2.10.95. RHEL 7.6 also uses Pango 1.42, making this inner check redundant.

Ubuntu 16.04 uses Pango 1.38, but fontconfig 2.11.94 (!). Any Ubuntu versions with a newer fontconfig will naturally have an equal or later Pango.

Given that this takes care of the two main Linux distributions in popular use, I think it's safe to remove this Pango version check.

Reply:
Removed the redundant Pango version check directive in 0330db5.